.github/workflows/doc-test.yaml

```diff
@@ -0,0 +1,42 @@
+name: Doctest
+
+# REQUIRED secrets used in this workflow:
+#   DOCKER_REGISTRY_TOKEN: token for USER user to sign into DOCKER_REGISTRY
+#
+# REQUIRED variables used in this workflow.
+#   GEOIPS_DEFAULT_BRANCH: Default branch on geoips repo (for checking out
+#                          geoips repo for tests and build scripts)
+#   DOCKER_REGISTRY: URL to the docker registry
+#   DOCKER_REGISTRY_USER: user to sign into DOCKER_REGISTRY
+#   RUNNER: default runner
+#   UPLOAD_GITHUB_ARTIFACTS: true/false, specify whether to upload artifacts to github
+#   USE_DOCKER_BUILDX_ACTION: true/false, specify whether to use the docker buildx
+#                             action. If false use explicit docker commands.
```

Comment:
doct-test, lint, and proper-release-note-edits included in template_basic_plugin since we assume people are often using the template repo from OUTSIDE the NRLMMD organization, so will need these local actions.